jdaviz/configs/mosviz/helper.py

```diff
@@ -547,7 +559,7 @@ def load_metadata(self, data_obj, ids=None, spectra=False, sp1d=False):
         self.app.load_data(data_obj, ids=ids, spectra=spectra, sp1d=sp1d,
                            parser_reference="mosviz-metadata-parser")
 
-    def load_1d_spectra(self, data_obj, data_labels=None):
+    def load_1d_spectra(self, data_obj, data_labels=None, **kwargs):
```

Comment:
I don't love all of these accepting `**kwargs` as it makes it easy for typos to slip through unnoticed... is there any reasonable alternative?  I think this is related to a previous question I had about `load_data` just directly accessing `self._default_....reference_name` instead of having them passed through all the methods as `kwargs`.